build-system/compile/post-closure-babel.js

```diff
@@ -73,16 +73,11 @@ async function terserMinify(code, filename) {
 
 /**
  * Apply Babel Transforms on output from Closure Compuler, then cleanup added
- * space with Terser. Used only in esm mode.
+ * space with Terser.
  * @param {string} file
  * @return {Promise<void>}
  */
 async function postClosureBabel(file) {
-  if ((!argv.esm && !argv.sxg) || path.extname(file) === '.map') {
-    debug(CompilationLifecycles['complete'], file);
```

Comment:
Does this subtly change the debug process for `.map` files?

Reply:
Oooh, excellent catch. We still need this block for `.map` files. Restored.